src/components/Breakpoint/Breakpoint.tsx

```diff
@@ -19,9 +20,11 @@ export const discriminate = (
   return { mode: 'gt' as ModeType, breakpoint: props.gt as BreakpointType };
 };
 
-export type BreakpointProps = At | GreaterThan | LessThan;
+export type BreakpointProps = (At | GreaterThan | LessThan) & {
+  className?: string;
```

Comment:
This doesn't feel right.  What's the use case?

Reply:
like here: https://github.com/ModaOperandi/discovery/compare/DOOR-296?expand=1#diff-af1a7dffc3b67455454466a72ecc183bR15-R16
that className in <div> has css like `position: sticky;` which gets ignored when i wrap it all with `<Breakpoint /> ` 

Not sure why, but applying `className="AboutPage__nav"` to Breakpoint instead should fix it , i think

Reply:
Also double-checked with using <Stack> instead of <div> there, and same thing.

Reply:
just saying, wouldn't be a soltion to do smth like 

<div className="AboutPage__nav">
    <Breakpoint gt="sm">
        ...
    </Breakpoint>
</div>

?

Reply:
yeah that works, thanks @robertmihaiman . 
I was just mostly curious why is it a bad idea to have an extra prop for className.

Reply:
it can break the meant behaviour, I guess;

and

it shouldn’t do more than rendering on specific breakpoint;